### PR TITLE
Add amadhusu to opendatahub-io, data-science-pipelines-operator, mode…

### PR DESCRIPTION
**Pull request by amadhusu**
…lmesh, modelmesh-runtime-adapter, modelmesh-serving, rest-proxy, odh-model-controller, data-science-pipelines

## New Open Data Hub Member Requirements
- [X] New members have reviewed and acknowledged the Open Data Hub:
  - [Code of Conduct](https://github.com/opendatahub-io/opendatahub-community/blob/main/CODE_OF_CONDUCT.md)
  - [Contribution Guide](https://github.com/opendatahub-io/opendatahub-community/blob/main/contributing.md)
  - [Community Membership Guidelines](https://github.com/opendatahub-io/opendatahub-community/blob/main/community-membership.md)
- [X] [Enabled 2FA on their GitHub account](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/about-two-factor-authentication)

## Pull Request Requirements
- [X] New members are added in alphabetical order
- [X] Only one new member change per commit (if you add two members separate it in two commits
- [X] For individual user changes: Commit message format `Add <USERNAME> to <opendatahub-io, mlops-sig, ...>`. 
- [ ] For new team requests: Commit message format `Create <TEAMNAME>`. If the new team consists solely of existing members, you may 
- [ ] New GitHub team requests are from an existing opendatahub-io member who will function as the maintainer of the team. Each additional member will follow the same requirements for adding new members

### CODE REVIEW

File: config/opendatahub-io/org.yaml [hunk 1]
```diff
@@ -177,6 +179,7 @@ orgs:
       repo-modelmesh-write:
         description: "Members with Write access to the modelmesh repository"
         members:
+        - amadhusu
```
Comment (accorvin) [inline config/opendatahub-io/org.yaml:182]: You don't need this access. I'm guessing you copied Humair's permissions, and his access here is an old artifact of a previous project he worked on.

File: config/opendatahub-io/org.yaml [hunk 2]
```diff
@@ -211,6 +214,7 @@ orgs:
       repo-modelmesh-runtime-adapter-write:
         description: "Members with Write access to the modelmesh-runtime-adapter repository"
         members:
+        - amadhusu
```
Comment (accorvin) [inline config/opendatahub-io/org.yaml:217]: You don't need this access. I'm guessing you copied Humair's permissions, and his access here is an old artifact of a previous project he worked on.

File: config/opendatahub-io/org.yaml [hunk 3]
```diff
@@ -245,6 +249,7 @@ orgs:
       repo-modelmesh-serving-write:
         description: "Members with Write access to the modelmesh-serving repository"
         members:
+        - amadhusu
```
Comment (accorvin) [inline config/opendatahub-io/org.yaml:252]: You don't need this access. I'm guessing you copied Humair's permissions, and his access here is an old artifact of a previous project he worked on.

File: config/opendatahub-io/org.yaml [hunk 4]
```diff
@@ -279,6 +284,7 @@ orgs:
       repo-rest-proxy-write:
         description: "Members with Write access to the rest-proxy repository"
         members:
+        - amadhusu
```
Comment (accorvin) [inline config/opendatahub-io/org.yaml:287]: You don't need this access. I'm guessing you copied Humair's permissions, and his access here is an old artifact of a previous project he worked on.

File: config/opendatahub-io/org.yaml [hunk 5]
```diff
@@ -313,6 +319,7 @@ orgs:
       repo-odh-model-controller-write:
         description: "Members with Write access to the odh-model-controller repository"
         members:
+        - amadhusu
```
Comment (accorvin) [inline config/opendatahub-io/org.yaml:322]: You don't need this access. I'm guessing you copied Humair's permissions, and his access here is an old artifact of a previous project he worked on.

**Review by accorvin — CHANGES_REQUESTED** [review]

File: config/opendatahub-io/org.yaml (hunk 1 above)
Comment (amadhusu) [reply config/opendatahub-io/org.yaml:182]: Yeah, I copied Humair's and Dharmit's permissions. I will undo these changes, thanks for the info.

File: config/opendatahub-io/org.yaml (hunk 2 above)
Comment (amadhusu) [reply config/opendatahub-io/org.yaml]: Yeah, I copied Humair's and Dharmit's permissions. I will undo these changes, thanks for the info.

File: config/opendatahub-io/org.yaml (hunk 3 above)
Comment (amadhusu) [reply config/opendatahub-io/org.yaml]: Yeah, I copied Humair's and Dharmit's permissions. I will undo these changes, thanks for the info.

**Review by accorvin — APPROVED** [review]

**Comment by accorvin** [comment]
@anishasthana can you look into this github action failure? At first glance it looks like a potential code issue to me.

**Comment by anishasthana** [comment]
I'll take a look!

**Review by anishasthana — CHANGES_REQUESTED** [review]
@amadhusu can you squash your commits?

**Comment by anishasthana** [comment]
@HumairAK can you ptal? Thansk!

**Review by HumairAK — APPROVED** [review]

**Comment by HumairAK** [comment]
small nit: git hub commit msg is now outdated as per the changes made, also it's good practice to keep first commit line under 72 char (most suggest 50 char or less), and not _too specific_ to avoid situations like this

**Pull request merged**